src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+* Up to 10 videos and screenshots of your app.
+* The production URL of your app.
+To get this URL, go to the app's associated *Project Details*, click *View Published Koji*, and copy the URL from the browser's address bar.
+
+To create your App Store listing,
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
+
+. Click *New App Template*.
+
+. Give the app template a name.
++
+This name is not published in the App Store; it is used to identify your listing.
+
+. Fill in the *App Template Details*.
++
+IMPORTANT: All texts and images in your listing must meet https://withkoji.com/resources/terms[Koji's Terms of Use] requirements.
++
+* *Icon* – The 500x500-pixel icon displayed with your listing.
+
+* *Name* – The title of your listing.
+Note that the title could be truncated depending on the width of the browser page; therefore, place distinguishing words near the beginning.
+
+* *Description* – Features and benefits of your app.
+
+* *Share Image* – The 1200x630-pixel thumbnail that is displayed when your app listing is shared in other social media sites.
+
+* *Videos and Screenshots* – Up to 10 videos and screenshots of your app.
+
+* *Bundled App* – The production URL of your app.
+
+* *Price to Remix* – The price (in US dollars) that the creator/influencer pays to publish their customized version of your app.
+Customization is free; creators/influencers are charged only when they publish their customized version in their Koji profile.
+You keep 85% of this amount; Koji gets 15%.
+Leave blank to allow the creator/influencer to publish their customized version for free.
+
+* *Transaction Fee Percent* – The percentage you wish to collect from each transaction if your app allows In-App Purchases.
+You keep 60% of this percentage; Koji gets 40%.
+Leave blank to collect the default percentage (15%).
++
+For example, if you set this to 10% and the user paid $10 in the app,
++
+** The creator/influencer who added your app to their profile receives $9.00.
+** You receive $0.60.
+** Koji receives $0.40.
+
+* *This template is open source.* – If enabled, other developers can view and fork your app's source code.
+
+* *This template is intended primarily for developers to fork.* – If enabled, your app listing will be placed in the *For Developers* category of the App Store.
+
+* *Variants* – (Optional) The listing URLs of other variations of the app.
+
+* *Attachments* – (Optional) Links to other sites that provide more information about your app, such as YouTube tutorials, news articles, and product comparisons or reviews.
+
+. Click btn:[Save and Publish].
+
+. https://forms.gle/exqnEVtC45Cizdw56[Fill out and submit the request form] to add your listing to the App Store.
++
+You will be notified when your listing is approved and posted in the App Store.
+
+NOTE: The version of your published listing is different from the published version of your app in your *Project Details*.
```

Comment:
This note is very confusing and seems a bit out of place here. We need to expand the information about updating app listings... possibly as another section on this page or possibly on a new page.  Most of the note probably belongs with that expanded content. The one part of the note that might be good in this procedure is the tidbit about each version having its own URL (which is something new/I didn't know had changed). If that's true, it might be a good note to add with the "Bundled App" instructions.

Reply:
If the website is changed to distinguish *app version* from *listing version*, we can get rid of this note.

I realized that there was a difference between the two after I saved the listing because it said "Version 1" when I knew that I already published 3 versions of my app. So I wrote the note because I figured someone else would also be confused.

I'm actually not sure about the production URL being different for each version. I just read somewhere that you need to update your listing if you create a new version of your app. I assumed it's because the URL changes. It needs to be tech-reviewed.

Reply:
I'm going to comment out the last two sentences for now:
> Each version of your app has a different production URL.
> You can update the listing to point to the latest app version, or you can create a new listing for the new app version to keep the old app version available for creators who might not want to upgrade.

Reply:
Ah, ok. AFAIK, the app is always published to the same URL (its permalink). However, the App Store continues to use the build version that was current when you created or updated your app listing (it doesn't access your app through the permalink). It won't automatically update the listing if you push another build. That lets a developer list their first working version on the store, and then continue to develop and deploy tests of future versions without affecting the working app on the store. When it's ready, they update the app listing. And, yes, we definitely need to distinguish between development build versions and App Store app versions. Maybe just referring to the development versions as "build numbers" will help here (and we can suggest UI text fixes to Sean, as well).

Reply:
I see a note that says, "This version has been published and can not be edited. To update this template, create a new version and publish that version." Then there's a *Create New Version* option on the left.

But in a separate Jira please. I think it opens up a can of worms (multiple scenarios).